pkg/table/tableinfo.go

```diff
@@ -106,6 +110,29 @@ func (t *TableInfo) setRowEstimate(ctx context.Context) error {
 	return nil
 }
 
+func (t *TableInfo) setIndexes(ctx context.Context) error {
```

Comment:
Is `Load` a better word here?, `set` (as in setter methods) to me has the connotation that you are passing in the value which will be stored in a private property of the object, here we are loading them from the DB and then `setting` them.

Not a big one, I am happy with either.

Reply:
There are existing other functions like `setRowEstimate`, `setPrimaryKey` that this follows. IIRC it was based on an earlier reviewer to use `set` here, since on some of the other features, load had ambiguous usage.